ovos_config/config.py

```diff
@@ -564,3 +363,16 @@ def clear_cache(message=None):
         """DEPRECATED - there is no cache anymore """
         Configuration.updated(message)
 
+
+def read_mycroft_config():
+    return dict(Configuration())
+
+
+def update_mycroft_config(config, path=None):
+    if path is None:
```

Comment:
Consider a docstring and type annotations (looks like `config` is a dict and not necessarily a `LocalConfig` object here)

Reply:
those are dict subclasses, this method expects `dict` i dont think clarification is necessary here, any dict subclass should work by definition